packages/web/src/components/edit/fields/price-and-audience/PriceAndAudienceField.tsx

```diff
@@ -610,7 +495,9 @@ export const AccessAndSaleField = (props: AccessAndSaleFieldProps) => {
         ...fieldVisibilityLabels
       ]
     } else {
-      selectedValues = [{ label: messages.public, icon: IconVisibilityPublic }]
+      selectedValues = isHiddenPaidScheduledEnabled
+        ? [{ label: messagesV2.public }]
```

Comment:
icon not needed?

Reply:
correct, just a "free for everyone" label